### PR TITLE
Changes home page to greet people by first name

### PR DESCRIPTION
**Pull request by carolwong492**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1284)

# List of squashed commits

- Added to tests

# Test Checklist

- [X] Unit Tests Passed:
- [X] Jasmine Tests Passed:
- [X] General Visual Inspection:

### CODE REVIEW

**Review by JorWo — APPROVED** [review]
Looks good! Thanks for adding the tests too.

File: src/main/java/edu/hawaii/its/groupings/access/User.java [hunk 1]
```diff
@@ -45,6 +45,10 @@ public String getName() {
         return attributes.getValue("cn");
     }
 
+    public String getGivenName() {
+        return attributes.getValue("givenName");
+    }
+
```
Comment (JorWo) [inline src/main/java/edu/hawaii/its/groupings/access/User.java:51]: Does getName() in User return the person's full name and getGivenName() return the first name? If so, can you change getGivenName() to getFirstName() to make it more clear.

Comment (fduckart) [reply src/main/java/edu/hawaii/its/groupings/access/User.java:51]: This one is on me. I was torn about this decision and suggested to Carol that we match up the getter with the underlying attribute in LDAP. I'm not sure what got into the LDAP people to name the attribute the way they did. Whatever you decide is good, I think.

Comment (fduckart) [reply src/main/java/edu/hawaii/its/groupings/access/User.java:51]: P.S. A lot of people (but not all) in the programming world have decided that firstname and/or lastname are considered one word, so need for uppercase on the letter N. If you go in for that line of thinking, of course.

Comment (JorWo) [reply src/main/java/edu/hawaii/its/groupings/access/User.java:51]: In that case, I'll keep it as is.

**Pull request merged**